Makefile

```diff
@@ -2,7 +2,7 @@
 
 CGO_ENABLED=0
 GOOS=linux
-CORE_IMAGES=./cmd/activator ./cmd/autoscaler ./cmd/autoscaler-hpa ./cmd/controller ./cmd/queue ./cmd/webhook ./vendor/knative.dev/pkg/apiextensions/storageversion/cmd/migrate
+CORE_IMAGES=./cmd/activator ./cmd/autoscaler ./cmd/autoscaler-hpa ./cmd/controller ./cmd/queue ./cmd/webhook ./cmd/default-domain ./pkg/cleanup/cmd/cleanup ./vendor/knative.dev/pkg/apiextensions/storageversion/cmd/migrate
```

Comment:
We might want to skip cleanup or remove the logic we have in S-O for resource cleanup cc @dsimansk, not going to block this PR.
Also I think we have the same in 1.15.